spec/tasks/build_db_spec.rb

```diff
@@ -1,70 +1,70 @@
-  	let(:all_nils) { 2 }
-
-    before(:all) do @bools = get_column_type(:boolean) end
-
-    it "sets fields to true when csv is 'Yes' or 'True'" do
-      row = Institution.find_by(facility_code: @csv_rows[all_yesses][:facility_code])
-      @bools.each { |bool| expect(row[bool]).to be_truthy }
-    end
-
-    it "sets fields to false when csv neither 'Yes' nor 'True'" do
-      row = Institution.find_by(facility_code: @csv_rows[all_nos][:facility_code])
-      @bools.each { |bool| expect(row[bool]).to be_falsy }
-    end
-
-    it "are false when csv is nil" do
-      row = Institution.find_by(facility_code: @csv_rows[all_nils][:facility_code])
-      @bools.each { |bool| expect(row[bool]).to be_falsy }
-    end
-  end
-
-  describe 'floats' do
-    before(:all) do @floats = get_column_type(:float) end
-
-    let(:all_valid_floats) { 3 }
-    let(:no_valid_floats) { 4 }
-    let(:valid_float_value) { 1234.5 }
-
-    it "valid strings are floats" do
-      row = Institution.find_by(facility_code: @csv_rows[all_valid_floats][:facility_code])
-      @floats.each { |float| puts "@@@@@@@@@@@@ #{float}"; expect(row[float]).to eq(valid_float_value) }
-      end
-
-    it "invalid strings are 0" do
+#    @csv_rows = CSV.read('rake_test_data.csv', headers: true, 
+#        encoding: "iso-8859-1:utf-8", header_converters: :symbol).map { |row| row.to_hash }
+
+#    silence_stream(STDOUT) do
+#      Rake::Task['load_csv'].invoke('rake_test_data.csv')
+#    end
+#  end
+
+#  describe 'transcribing Csv to DB' do
+#    it 'both have the same row count' do 
+#      expect(@csv_rows.length).to eql(Institution.count)
+#    end
+#  end
+
+#  describe 'processing booleans' do
+#  	let(:all_yesses) { 0 }
+#  	let(:all_nos) { 1 }
+#  	let(:all_nils) { 2 }
+
+#    before(:all) do @bools = get_column_type(:boolean) end
+
+#    it "sets fields to true when csv is 'Yes' or 'True'" do
+#      row = Institution.find_by(facility_code: @csv_rows[all_yesses][:facility_code])
+#      @bools.each { |bool| expect(row[bool]).to be_truthy }
+#    end
+
+#    it "sets fields to false when csv neither 'Yes' nor 'True'" do
+#      row = Institution.find_by(facility_code: @csv_rows[all_nos][:facility_code])
+#      @bools.each { |bool| expect(row[bool]).to be_falsy }
+#    end
+
+#    it "are false when csv is nil" do
+#      row = Institution.find_by(facility_code: @csv_rows[all_nils][:facility_code])
+#      @bools.each { |bool| expect(row[bool]).to be_falsy }
+#    end
+#  end
+
+#  describe 'floats' do
+#    before(:all) do @floats = get_column_type(:float) end
+
+#    let(:all_valid_floats) { 3 }
+#    let(:no_valid_floats) { 4 }
+#    let(:valid_float_value) { 1234.5 }
+
+#    it "valid strings are floats" do
+#      row = Institution.find_by(facility_code: @csv_rows[all_valid_floats][:facility_code])
+#      @floats.each { |float| puts "@@@@@@@@@@@@ #{float}"; expect(row[float]).to eq(valid_float_value) }
+#      end
+
+#    it "invalid strings are 0" do
```

Comment:
Any reason why all these commented out lines can't be deleted (and recovered if need be via Git)